mobify/pinny
# Iframe test

Iframe test harness for unit testing
  
Status: **Ready to merge**
 
Reviewers: @scalvert 
JIRA: https://mobify.atlassian.net/browse/CSOPS-1096?filter=-1
  
## Changes
- (change1)
  
## How to test-drive this PR
- run grunt test:browser
- observe that each pinny is initialized within separate iframes to avoid testing contamination
